services/management/mongodb.go

```diff
@@ -72,6 +72,22 @@ func (s *MongoDBService) Add(ctx context.Context, req *managementpb.AddMongoDBRe
 
 		mongoDBOptions := models.MongoDBOptionsFromRequest(req)
 
+		if req.MetricsMode == managementpb.MetricsMode_AUTO {
```

Comment:
Can we extract this block to some function, since we use it in 4 places?

Reply:
Yes sure.